src/Quartz.Extensions.DependencyInjection/ServiceCollectionQuartzConfigurator.cs

```diff
@@ -61,15 +61,10 @@ public void UseMicrosoftDependencyInjectionJobFactory(Action<JobFactoryOptions>?
             UseJobFactory<MicrosoftDependencyInjectionJobFactory>(configure);
         }
 
+        [Obsolete("Jobs are always created with some")]
```

Comment:
Did you mean `... with scope`?

Reply:
Thanks, working in parallel with two bug fixes seems to freeze the brain. Pushed fix commit.